docs/user-guide/timeline.md

```diff
@@ -1,114 +1,90 @@
+The **Step backward** button <img src="../img/button/timeline-step-backward-button.jpg" class="ms-docbutton"/> and the **Step forward** button <img src="../img/button/timeline-step-forward-button.jpg" class="ms-docbutton"/> allow the user to statically change the current time, so by clicking on one of them the cursor changes its position (to the previous or the next step) on the histogram and the date/time values of the control cells will be update accordingly.
 <br>
-You can pause the animation with the **Pause** button <img src="../img/button/timeline-pause-button.jpg" class="ms-docbutton"/>.
+The user can pause the animation through the **Pause** button <img src="../img/button/timeline-pause-button.jpg" class="ms-docbutton"/>, as follows:
 
 <img src="../img/timeline/timeline-animation.gif" class="ms-docimage"  style="max-width:700px;" />
 
-You can also specify a *time range*. During the animation, the whole range will be shifted step by step along the time axis and in each step the layers' configurations will be consider the total amount of data in that range.
 
-<img src="../img/timeline/timeline-animation-range.gif" class="ms-docimage"  style="max-width:700px;"/>
+!!! Note
+    The user can also specify a *time range*. During the animation, the whole range will be shifted step by step along the time axis and in each step the layer configuration will be consider the total amount of data in that range.
+    <img src="../img/timeline/timeline-animation-range.gif" class="ms-docimage"  style="max-width:700px;"/>
 
 ### Animation Settings
 
-The animation behavior can be customized through the **Settings**. If you click on the button <img src="../img/button/timeline-playback-settings-button.jpg" class="ms-docbutton"/> a panel with two different sections appears.
+The animation behavior can be customized through the **Settings** button <img src="../img/button/timeline-playback-settings-button.jpg" class="ms-docbutton"/> allowing the user to setting the *Timeline* and the *Playback*. 
 
 <img src="../img/timeline/timeline-animation-settings.jpg" class="ms-docimage"  style="max-width:500px;" />
 
-***Timeline Settings***
+By default, the **Snap to guide layer** is enabled allows to force the time cursor to snap to the selected layer's data. The user can disable *Snap to guide level* to select the time cursor he prefers through the section **Animation passage**, for example, the process could be similar to the following: 
 
-* **Snap to guide layer**: this option is active by default. It forces the time cursor to snap to the selected layer's data. Disabling this option the **Animation Step** selection in the ***Playback Settings*** section becomes available.
+<img src="../img/timeline/animation-passage.jpg" class="ms-docimage"/>
 
-***Playback Settings***
+The user can set the second between one frame and another through the **Frame Duration** and enable the **Follow the animation** to visualize the animation also on the histogram.
```

Comment:
The user can set the number of second between one animation frame and another through the **Frame Duration** and enable the **Follow the animation** to visualize the animation process also inside the histogram: the histogram will automatically move to follow the animation.